ui/pages/get-mooney.tsx

```diff
@@ -59,35 +43,35 @@ export default function Join({ usdQuotes }: any) {
   )
 }
 
-export async function getStaticProps() {
-  const DAI = new Token(
-    137,
-    DAI_ADDRESSES['polygon'],
-    18,
-    'DAI',
-    'DAI Stablecoin'
-  )
+// export async function getStaticProps() {
```

Comment:
I get the idea of just having this commented out - but I wonder if we can archive it somewhere else in the app instead? Not a good practice to have commented out code sitting around in the codebase.